examples/templates/gcp-devcontainer/main.tf

```diff
@@ -6,16 +6,49 @@ terraform {
     google = {
       source = "hashicorp/google"
     }
+    envbuilder = {
+      source = "coder/envbuilder"
+    }
   }
 }
 
 provider "coder" {
 }
 
+provider "google" {
+  zone    = data.coder_parameter.zone.value
+  project = var.project_id
+}
+
+data "google_compute_default_service_account" "default" {}
+
+data "coder_workspace" "me" {
+}
+data "coder_workspace_owner" "me" {}
+
 variable "project_id" {
   description = "Which Google Compute Project should your workspace live in?"
 }
 
+variable "cache_repo" {
+  default     = ""
+  description = "(Optional) Use a container registry as a cache to speed up builds."
+  type        = string
+}
+
+variable "insecure_cache_repo" {
```

Comment:
This changeset is relatively big, so is the `.tf` file. Maybe wipe out this variable?